File: python/arrays/max_element.py
n = int(input("Enter size of array: "))
arr = list(map(int, input(f"Enter {n} elements: ").split()))

max_val = arr[0]
for num in arr[1:]:
    if num > max_val:
        max_val = num

print("Maximum element:", max_val)

File: java/arrays/max_element.java
public class MaxElement {
    public static void main(String[] args) {
        int[] arr = {3, 7, 1, 9, 4};

        if (arr.length == 0) {
            System.out.println("Array is empty!");
            return;
        }

        int max = arr[0]; // assume first element is max

        for (int i = 1; i < arr.length; i++) {
            if (arr[i] > max) {
                max = arr[i]; // update max if current element is bigger
            }
        }

        System.out.println("Maximum element is: " + max);
    }
}
